velox/docs/functions/spark/aggregate.rst

```diff
@@ -81,4 +81,11 @@ General Aggregate Functions
     Returns the value of `x` associated with the minimum value of `y`.
     Note: Spark provides a non-strictly comparator which is less than or equals to.
     Eg. SELECT min_by(x, y) FROM VALUES (('a', 10)), (('b', 10)), (('c', 50)) AS tab(x, y);
-        > b
\ No newline at end of file
+        > b
+
+.. spark:function:: sum(x) -> x
+
+    Returns the sum of `x`.
+    Note: When the sum of bigint values exceeds its limit, it cycles to the overflowed value rather than raising an error.
+    Eg. SELECT SUM(x) FROM (VALUES (9223372036854775807L), (1L)) AS tab(x);
```

Comment:
Please, generate the docs and verify proper formatting.

See examples in https://facebookincubator.github.io/velox/functions/presto/aggregate.html#set_union

Make sure to fix other instances in this file.